cabal.GHC-8.8.3.config

```diff
@@ -44,7 +44,7 @@ constraints: any.Cabal ==3.0.1.0,
              any.blaze-html ==0.9.1.2,
              any.blaze-markup ==0.8.2.7,
              any.boomerang ==1.4.6,
-             any.bv-sized ==1.0.1,
+             any.bv-sized ==1.0.2,
```

Comment:
Does it make sense to keep the 8.8.3 freeze file around at all if we're migrating to 8.8.4?

Reply:
I just wanted to avoid breaking anything. I think we should remove it ASAP.